tools/cpp/unix_cc_toolchain_config.bzl

```diff
@@ -96,6 +96,92 @@ def layering_check_features(compiler):
         ),
     ]
 
+def interface_shared_libraries_support(cc_path, llvm_ifs_path, link_dynamic_library_tool):
+    if not llvm_ifs_path:
+        return [], []
+    dynamic_library_actions = [
+        ACTION_NAMES.cpp_link_dynamic_library,
+        ACTION_NAMES.cpp_link_nodeps_dynamic_library,
+        ACTION_NAMES.lto_index_for_dynamic_library,
+        ACTION_NAMES.lto_index_for_nodeps_dynamic_library,
+    ]
+    action_configs = [
+        action_config(
+            action_name = action,
+            tools = [tool(tool = link_dynamic_library_tool)],
```

Comment:
When using the `tool` attribute, do I still have to manually add the tool to the toolchain files? The sandboxed test passes, but I'm not entirely sure how the toolchain picks up this file with `has_configured_linker_path`.